packages/lib/src/core/Errors/utils.test.ts

```diff
@@ -3,9 +3,10 @@ import { mapFieldKey as mapFieldKeyCC } from '../../components/Card/components/C
 import Language from '../../language';
 import { SortErrorsObj } from './types';
 import { sortErrorsByLayout } from './utils';
+import enUS from '../../../../server/translations/en-US.json';
 
 describe('Tests for Errors/utils', () => {
-    const i18n = new Language('en-US');
+    const i18n = new Language({ locale: 'en-US', translations: enUS });
```

Comment:
This could just be `const i18n = global.i18n;`
...since you've done such a nice job tidying up! :-)

Reply:
I will try